integration_test/cases/browser/send_text_to_active_element_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Wallaby.Integration.Browser.SendTextToActiveElementTest do
```

Comment:
Can we rename it to `send_keys` from `send_text`? This one and the other existing test in browser, not sure why it was `send_text`

Reply:
OK, sure!

Reply:
Done. :)

Reply:
Awesome, can you also rename `integration_test/cases/browser/send_text_test.exs` to `integration_test/cases/browser/send_keys_test.exs` while at it?

Reply:
Also done, @michallepicki